### PR TITLE
feat(alert): shim and fallbacks removal 

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-1215

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/angular/src/data/tree-view/_variables.tree-view.scss [hunk 1]
```diff
@@ -27,7 +28,9 @@ $clr-tree-link-hover-color: nav-variables.$clr-custom-links-hover-color !default
 $clr-tree-link-selection-color: variables.$clr-global-selection-color !default;
 $clr-tree-link-text-color: variables.$clr-color-neutral-700 !default;
 $clr-tree-checkbox-padding-top: ($clr-tree-node-touch-target - variables.$clr_baselineRem_1) * 0.5 !default;
-$clr-tree-checkbox-padding-left: ($clr-tree-node-touch-target - alert-variables.$clr-icon-dimension-sm) * 0.5 !default;
+$clr-tree-checkbox-padding-left: calc(
+  calc(#{$clr-tree-node-touch-target} - #{alert-variables.$clr-icon-dimension-sm}) * 0.5
+) !default;
```
Comment (dtsanevmw) [inline projects/angular/src/data/tree-view/_variables.tree-view.scss:33]: Try to use single `calc` with parentheses. It should be identical.

```suggestion
$clr-tree-checkbox-padding-left: calc((#{$clr-tree-node-touch-target} - #{alert-variables.$clr-icon-dimension-sm}) * 0.5
) !default;
```

File: projects/angular/src/data/tree-view/_tree-view.clarity.scss [hunk 2]
```diff
@@ -177,7 +177,9 @@
   .clr-treenode-spinner-container {
     //Dimensions
     @include mixins.equilateral(tree-view-variables.$clr-tree-node-touch-target);
-    padding: (tree-view-variables.$clr-tree-node-touch-target - tree-view-variables.$clr-tree-spinner-size) * 0.5;
+    padding: calc(
+      calc(#{tree-view-variables.$clr-tree-node-touch-target} - #{tree-view-variables.$clr-tree-spinner-size}) * 0.5
+    );
```
Comment (dtsanevmw) [inline projects/angular/src/data/tree-view/_tree-view.clarity.scss:182]: Try to use single calc with parentheses. It should be identical.
```suggestion
    padding: calc((#{tree-view-variables.$clr-tree-node-touch-target} - #{tree-view-variables.$clr-tree-spinner-size}) * 0.5
    );
```

File: projects/angular/src/emphasis/alert/_alert.clarity.scss [hunk 3]
```diff
@@ -11,272 +11,138 @@
...
-      &:focus,
-      &:hover,
-      &:active {
-        @include mixins.css-var(
-          border-color,
-          $myActionActiveCssVar,
-          $myActionActiveSassVar,
-          variables.$clr-use-custom-properties
-        );
+        color: $myActionSassVar;
```
Comment (dtsanevmw) [inline projects/angular/src/emphasis/alert/_alert.clarity.scss:54]: Shouldn't this be different for 'focus,hover,active' ?

Comment (valentin-mladenov) [reply projects/angular/src/emphasis/alert/_alert.clarity.scss:54]: Alert action buttons will follow `outline button` specification. 

Active is equal to regular `button outline` colors.
Hover is handled by `@extend .btn-{alertType}-outline;` lines in  `.alert-app-level { ... }` section.

File: projects/angular/src/data/tree-view/_tree-view.clarity.scss (hunk 2 above)
Comment (valentin-mladenov) [reply projects/angular/src/data/tree-view/_tree-view.clarity.scss]: corrected in commit 3332a98

File: projects/angular/src/data/tree-view/_variables.tree-view.scss (hunk 1 above)
Comment (valentin-mladenov) [reply projects/angular/src/data/tree-view/_variables.tree-view.scss]: corrected in commit 3332a98

**Review by dtsanevmw — APPROVED** [review]

**Pull request merged**